managers/parsers/inTechOpenParser.py

```diff
@@ -23,35 +23,39 @@ def createLinks(self):
 
             manifestJSON = self.generateManifest(self.uri, manifestURI)
 
+            htmlURI = self.createHTMLURI()
+
+            # TODO
+            # 1. Add regex to extract page identifier from PDF (e.g. r'intechopen.com\/books\/([\d]+)')
+            # 2. Construct HTML URI using extracted identifier
+            # 3. Add HTML URI to return array below
+            #
+            # QUESTIONS
+            # 1. Do all Intechopen books have PDF links?
+            # 2. How do we handle potential errors/add a fallback condition
+            # 3. Are PDF links formatted consistently?
+
             return [
                 (manifestURI, {'reader': True}, 'application/webpub+json', (manifestPath, manifestJSON), None),
-                (self.uri, {'reader': False, 'download': True}, self.mediaType, None, None)
+                (self.uri, {'reader': False, 'download': True}, self.mediaType, None, None),
+                (htmlURI, {'reader': False}, 'text/html', self.mediaType, None, None)
```

Comment:
`self.mediaType` is duplicative of the explicit media type set here so we should delete it!